core/views.py

```diff
@@ -0,0 +1,43 @@
+from django.http import HttpResponse, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.parsers import JSONParser
+from core.models import Book
+from core.serializers import BookSerializer
+# Create your views here.
+
+@csrf_exempt
+def book_list(request):
+    if request.method == 'GET':
+        books = Book.objects.all()
+        serializer = BookSerializer(books, many=True)
+        return JsonResponse(serializer.data, safe=False)
+
+    elif request.method == 'POST':
+        data = JSONParser().parse(request)
+        serializer = BookSerializer(data=data)
+        if serializer.is_valid():
+            serializer.save()
+            return JsonResponse(serializer.data, status=201)
+        return JsonResponse(serializer.errors, status=400)
+@csrf_exempt
+def book_detail(request, pk):
```

Comment:
this method also handled multiple HTTP methods, each HTTP method should get their own view